assets/src/edit-story/utils/processPastedElements.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { v4 as uuidv4 } from 'uuid';
+
+const DOUBLE_DASH_ESCAPE = '_DOUBLEDASH_';
+
+/**
+ * Processes pasted content to find story elements.
+ *
+ * @param {string} content
+ * @param {Object} currentPage
+ * @return {[]} Array of found elements.
+ */
+function processPastedElements(content, currentPage) {
+  let foundElements = [];
+  for (let n = content.firstChild; n; n = n.nextSibling) {
```

Comment:
It's really nice to have `copy` and `paste` functions in one module: it's not too big, but there's a clear connection between the output and input formats.